packages/xo-web/src/common/xo/index.js

```diff
@@ -615,6 +615,35 @@ export const addHostsToPool = pool =>
     })
   })
 
+export const enableAdvancedLiveTelemetry = async host => {
+  const isConfiguredToReceiveStreaming = await _call(
+    'netdata.isConfiguredToReceiveStreaming',
+    { host: host.id }
+  )
+  if (!isConfiguredToReceiveStreaming) {
+    await _call('netdata.configureXoaToReceiveData', {})
+  }
+  await _call('netdata.configureHostToStreamHere', {
+    host: host.id,
+  })
+  success(_('advancedLiveTelemetry'), _('enableAdvancedLiveTelemetrySuccess'))
+}
+
+export const isNetDataInstalledOnHost = async host => {
+  const isNetDataInstalledOnHost = await _call(
+    'netdata.isNetDataInstalledOnHost',
+    { host: host.id }
+  )
+  const hostApiKey = await _call('netdata.getHostApiKey', {
+    host: host.id,
+  })
+  const localApiKey = await _call('netdata.getLocalApiKey', {})
+  if (Boolean(isNetDataInstalledOnHost) && hostApiKey === localApiKey) {
```

Comment:
Why this cast?

Reply:
The api returns a `string` instead of a boolean

Reply:
Do a more explicit test please.

Reply:
Why don't you need a cast anymore?

Reply:
Because I opened a merge request in XOA project as requested in order to not cast values here